tests/ci/azure_pipeline_test/dsvm_nightly_linux_cpu.yml

```diff
@@ -31,18 +31,16 @@ jobs:
       python ./scripts/generate_conda_file.py --name nightly_reco_base && \
       conda env create --quiet -f nightly_reco_base.yaml 2> log
     displayName: 'Setup Conda Env'
-    timeoutInMinutes: 10
 
   - script: |
       . /anaconda/etc/profile.d/conda.sh && \
       conda activate nightly_reco_base && \
       echo "Smoke tests" && \
-      pytest tests/smoke -m "smoke and not spark and not gpu" --junitxml=reports/test-smoke.xml && \
+      pytest tests/smoke --durations 0 -m "smoke and not spark and not gpu" --junitxml=reports/test-smoke.xml && \
```

Comment:
when using the "script: |" action, you do not need to include && \ on every line, and can instead remove all of these (unless they are specifically desired).
This seems like legacy from pre-yml ADO